### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by harveysun321** [getlantern/lantern-forum-cn#1025] [General]
APP打开很卡，网络连接不稳定，无法正常使用，请求退款，谢谢。邮箱：604311570@qq.com，以下是订单详情：
 
广州饶铭网络科技有限公司
-617.00
交易成功
支付时间
2024-11-07 01:23:47
付款方式
中信银行信用卡
商品说明
游戏充值订单号P01202411070120541435554

收款方全称
广州饶铭网络科技有限公司
订单号
2024110722001424431456094281
商家订单号
2024110701205930669

**Comment by MaxMKMax** [comment]
@harveysun321 您好. 已经帮您提交好了. 请您耐心等待. 谢谢.

**Reply by harveysun321** [reply]
大概还要多长时间

**Reply by MaxMKMax** [reply]
@harveysun321 您好. 应该 2 周左右吧

**Reply by harveysun321** [reply]
> @harveysun321 您好. 应该 2 周左右吧

你好，请帮我看一下退款情况，已经过一个月了

**Reply by harveysun321** [reply]
> @harveysun321 您好. 已经帮您提交好了. 请您耐心等待. 谢谢.

你好，请帮我看一下退款情况，已经一个多月了

**Comment by MaxMKMax** [comment]
@harveysun321 您的专业版应该可以恢复了. 不好意思让您久等了.
